src/cls/_ZPM/Main.cls

```diff
@@ -1602,9 +1593,9 @@ ClassMethod Manage(ByRef pCommandInfo) [ Internal ]
 		// TODO: see if there is anything installed in this namespace.
 	}
 	
-	Set tNSMenu($i(tNSMenu)) = "Install an application"
-	Set tNSMenu($i(tNSMenu)) = "Upgrade/repair an application"
-	Set tNSMenu($i(tNSMenu)) = "Uninstall an application"
```

Comment:
This entire method can be removed instead as it is specific to Application packaging instead of Module. Also remove the corresponding reference in ShellInternal() and the command from the XData block.